models/engine/db_storage.py

```diff
@@ -74,3 +74,29 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+
+    def get(self, cls, id):
+        """A method to retrieve one object"""
+        if cls not in classes.values():
+            return None
+        all_values = models.storage.all(cls)
+        for value in all_values().values():
```

Comment:
When you using the for check the all_values() since it is a dict  you don't need to use the parentheses